### PR TITLE
Add function and utility for listing all available plugins.

### PR DESCRIPTION
**Pull request by rtibbles**
### Summary
Adds a plugin utility function and a CLI wrapper command to list all Kolibri plugins available on the local system.

### Reviewer guidance
Does it correctly list all Kolibri core plugins?
Does it correctly list all locally installed external plugins?
Does the form of the manifest file make sense?

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by rtibbles** [comment]
Will sort out the test failure.

File: kolibri/kolibri_plugin_manifest.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+plugins = []
+for item in resource_listdir("kolibri.plugins", "."):
+    try:
+        if resource_isdir("kolibri.plugins", item) and resource_exists(
+            "kolibri.plugins." + item, "kolibri_plugin.py"
+        ):
+            plugins.append("kolibri.plugins." + item)
+    except ImportError:
+        pass
+    except NotImplementedError:
```
Comment (jredrejo) [inline kolibri/kolibri_plugin_manifest.py]: I have added these two lines (I want to add them as a suggestion but it seems GH has allowed me to commit it :O )

**Review by jredrejo — CHANGES_REQUESTED** [review]
- I have seen that `kolibri plugin list` lists plugins that are disabled too, is it intended?
- I know It's a minor code change, but there are no tests at all for the two new functions

**Comment by rtibbles** [comment]
Yep, happy to add a test.

It lists all plugins that are available, so that you know which ones you can or can't disable. Could add their current status to the read out if helpful!

**Comment by jredrejo** [comment]
> Yep, happy to add a test.
> 
> It lists all plugins that are available, so that you know which ones you can or can't disable. Could add their current status to the read out if helpful!

I think so, people could be confused seeing a plugin in the list that's not doing anything. If the state is shown, it's very clear.

**Review by jredrejo — APPROVED** [review]
Tests pass, and the output is really useful:
```
$ kolibri plugin list


Available plugins                   Status
kolibri.plugins.coach               ENABLED
kolibri.plugins.default_theme       ENABLED
kolibri.plugins.device              ENABLED
kolibri.plugins.epub_viewer         ENABLED
kolibri.plugins.facility            ENABLED
kolibri.plugins.html5_viewer        ENABLED
kolibri.plugins.learn               ENABLED
kolibri.plugins.media_player        ENABLED
kolibri.plugins.pdf_viewer          ENABLED
kolibri.plugins.setup_wizard        ENABLED
kolibri.plugins.slideshow_viewer    ENABLED
kolibri.plugins.style_guide         ENABLED
kolibri.plugins.user                ENABLED
kolibri_exercise_perseus_plugin     ENABLED
kolibri_oidc_client_plugin          ENABLED
kolibri_opensearch_plugin           ENABLED
```

**Pull request merged**